### PR TITLE
[Fortran/gfortran][NFC] Update README.md

### PR DESCRIPTION
**Pull request by tarunprabhu**
Remove references to the "old" way of doing things which has now been removed. Add text describing the correct handling of directives. Add notes describing some other limitations that would be good to remove.

### CODE REVIEW

**Review by tblah — APPROVED** [review]
Thanks!

File: Fortran/gfortran/README.md [hunk 1]
```diff
@@ -227,15 +225,28 @@ the file for the presence or absence of certain text. To capture the same
...
 
-The `target` directive is used to restrict tests to run on certain
-platforms/systems. Currently, the target directive is ignored entirely and the
-tests are always run. Currently, the gfortran tests are only enabled on *nix on
-x86-64 and aarch64 and ignoring the directive seems to be ok. As support for
-more systems and architectures are added, these directives will need to be
-handled correctly.
+Some tests fail on certain platforms but not on others. There is, currently, no
+way to disable these tests on a specific platform and these are disabled 
+everywhere. This is obviously not ideal since we the extra coverage that the
```
Comment (luporl) [inline Fortran/gfortran/README.md:250]:
```suggestion
everywhere. This is obviously not ideal since the extra coverage that the
```

**Review by luporl — APPROVED** [review]
LGTM, thanks!

File: Fortran/gfortran/README.md (hunk 1 above)
Comment (tarunprabhu) [reply Fortran/gfortran/README.md]: Done. Thanks!

**Pull request merged**